doc/nrf/libraries/dfu/dfu_multi_image.rst

```diff
@@ -0,0 +1,41 @@
+.. _lib_dfu_multi_image:
+
+DFU multi-image
+###############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The DFU multi-image library provides an API for writing a DFU multi-image package that is being downloaded in chunks of an arbitrary size.
+The DFU multi-image package is a simple archive file that consists of a CBOR-based header that describes contents of the package, followed by a number of updates images, such as firmware images for different MCU cores.
+Images included in a DFU multi-image package are identified by numeric identifiers assigned by the user.
+The library provides a way for the user to register custom functions for writing a single image with a given identifier.
+Given that the library makes no assumptions with respect to formats of images included in a written package, it makes it a general-purpose solution for device firmware upgrades.
```

Comment:
```suggestion
Because the library makes no assumptions about the formats of images included in a written package, it serves as a general-purpose solution for device firmware upgrades.
```